plugin/go.vim

```diff
@@ -192,6 +191,8 @@ function! s:GoInstallBinaries(updateBinaries, ...)
         endif
 
         " GO111MODULE must be off to install gometalinter.
+        " TODO: determine if this line is still needed now that gometalinter
+        " has been depreciated
```

Comment:
Please leave this line in, but we can do with out the TODO _and_ the preceding line now.

Reply:
You mean delete my TODO line and the original `GO111MODULE` comment line? Done.